src/components/EvModuleInfo.vue

```diff
@@ -7,14 +7,16 @@
       <icon-button-with-tooltip
           icon="mdi-close"
           title="Discard selection"
+          variant="text"
+          density="compact"
           @on-click="context.unselect()"
       />
     </template>
     <v-card-text>
-      <p class="font-weight-bold">Module type: {{ module_node.instance.type }}</p>
+      <p class="font-weight-bold">Module type: {{ module_node?.instance?.type }}</p>
```

Comment:
Ich gehe davon aus, dass du mit strictNullCheck angefangen hast, aber das nicht umsetzbar war?

Reply:
The whole codebase is not yet ready for strictNullCheck. We should refactor it to do strictNullChecking over time, but currently this is less important.